backend/store/playlist.go

```
@@ -0,0 +1,170 @@
+
+func (ps PlaylistStore) RemoveSongFromPlaylist(playlistId uuid.UUID, songId uuid.UUID) error {
+	_, err := ps.db.Exec("DELETE FROM playlist_song WHERE playlist_id = $1 AND song_id = $2", playlistId, songId)
+	return err
+}
+
+func (ps PlaylistStore) RemovePlaylist(playlistId uuid.UUID, userId uuid.UUID) error {
+	tx, err := ps.db.Beginx()
+	if err != nil {
+		return err
+	}
+	defer tx.Rollback()
```

golangci-lint (1.62.2)

Error return value of `tx.Rollback` is not checked

(errcheck)